src/utils/isValidZipCode.js

```diff
@@ -0,0 +1,11 @@
+const ZIP_COUNT = 48073;
+
+/**
+ * Check if a given integer is a valid zip code (between 1000 and 48073)
+ * @param {Number} code zip code
+ * @returns {Boolean}
+ */
+const isValidZipCode = (code) =>
+  Number.isInteger(code) && code >= 1000 && code <= ZIP_COUNT;
```

Comment:
Your point in #24 is valid IMO, we can pass a string like `'01027'`, so I'd suggest either just checking if code is not `NaN` (with [Number.isNaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN)), or you use `parseInt(code)` then you check for `parsedCode >= 1000 && parsedCode <= ZIP_COUNT;`